### PR TITLE
runfix: address broken inputs when removing devices acc-210

### PR DESCRIPTION
**Pull request by V-Gira**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/ACC-210" title="ACC-210" target="_blank">BugACC-210</a>  [web] Usability of deleting a device on login (when device limit is reached) can be improved
  </td></table>
  <br />
 

----

# What's new in this PR?

### Issues

- Inputs in the remove device screen are way too narrow
3306c1a8-46b6-4250-b961-df51e37a355d

### Solutions

- adjust style according to [new mockup](https://www.figma.com/file/OrP9jAJdfttfvsm3nxnihP/Accessibility-Quick-Wins-(Web%2FDesktop)?node-id=2711%3A17289)
- (Changes to inputs border radius need to be adressed in the ui-kit)

image

### CODE REVIEW

**Review by PatrykBuniX — APPROVED** [review]

**Review by PatrykBuniX — APPROVED** [review]

File: src/script/auth/component/ClientItem.tsx [hunk 1]
```diff
@@ -186,9 +186,10 @@ const ClientItem = ({selected, onClientRemoval, onClick, client, clientError, re
   };
 
   const animationPosition = animationStep / CONFIG.animationSteps;
-  const height = animationPosition * 56;
+  const height = animationPosition * 70;
```
Comment (przemvs) [inline src/script/auth/component/ClientItem.tsx]: What's 70 mean? Move this to variable, also please move another magic numbers :D

**Review by przemvs — CHANGES_REQUESTED** [review]
LGTM, 1 change :D

File: src/script/auth/component/ClientItem.tsx (hunk 1 above)
Comment (V-Gira) [reply src/script/auth/component/ClientItem.tsx]: Dammit, someone noticed! I'll have to clean up that mess :face_with_spiral_eyes:  I'll move all the paddings, margins, etc to variables. I plead non guilty though

File: src/script/auth/component/ClientItem.tsx [hunk 2]
```diff
@@ -197,8 +208,8 @@ const ClientItem = ({selected, onClientRemoval, onClick, client, clientError, re
           ['&:focus-within']: {
             boxShadow: `0 0 0 1px ${COLOR.BLUE}`,
           },
-          backgroundColor: selected ? 'white' : '',
-          borderRadius: '4px',
+          backgroundColor: selected ? '#FFF' : '',
```
Comment (arjita-mitra) [inline src/script/auth/component/ClientItem.tsx:211]: can you use @white instead? its in color.less

Comment (V-Gira) [reply src/script/auth/component/ClientItem.tsx:211]: Don't have access to those css variables here I'm afraid :(

File: src/script/auth/component/ClientItem.tsx [hunk 3]
```diff
@@ -185,10 +185,21 @@ const ClientItem = ({selected, onClientRemoval, onClick, client, clientError, re
     setIsValidPassword(true);
   };
 
+  const spacing = {
```
Comment (przemvs) [inline src/script/auth/component/ClientItem.tsx]: It's bad name for variable. Please change it for what it mean e.g inputSpacing, sthSpacing.

**Review by przemvs — CHANGES_REQUESTED** [review]
1 thing left :)

File: src/script/auth/component/ClientItem.tsx (hunk 3 above)
Comment (V-Gira) [inline src/script/auth/component/ClientItem.tsx]:
```suggestion
  const animatedCardSpacing = {
```

**Review by przemvs — APPROVED** [review]

**Pull request merged**